src/com/company/SearchTheWord.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+import java.sql.SQLException;
+import java.util.StringTokenizer;
+
+import static com.company.ErrorMessage.SUCCESS_MESSAGE_WORD_FOUND;
+import static com.company.Main.*;
+
+public class SearchTheWord extends Thread {
+    String data;
```

Comment:
Make all the fields private. Line 10 to Line 13

Reply:
Not addressed?

Reply:
changed with that